actions/datastore_get.py

```diff
@@ -0,0 +1,86 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from vmwarelib import inventory
+from vmwarelib.serialize import DatastoreGetJSONEncoder
+from vmwarelib.actions import BaseAction
+from pyVmomi import vim  # pylint: disable-msg=E0611
+import json
+
+
+class DatastoreGet(BaseAction):
+    def get_all(self):
+        results = []
+        datastores = inventory.get_managed_entities(self.si_content, vim.Datastore)
+        for datastore in datastores.view:
+            result_dict = {
+                'name': datastore.name,
+                'summary': json.loads(json.dumps(datastore.summary, cls=DatastoreGetJSONEncoder))
```

Comment:
looks like this "result_dict" code is copy/pasted a few times, could probably make it a function that accepts a `datastore` and returns a `dict`